tests/PHPUnit/Framework/Mock/Concurrency/LockBackend/InMemoryLockBackend.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Tests\Framework\Mock\Concurrency\LockBackend;
+
+use Piwik\Concurrency\LockBackend;
+
+/**
+ * Lock Backend that stores locks in memory only
+ */
+class InMemoryLockBackend implements LockBackend
+{
+    public $locks = [];
+
+    public function getKeysMatchingPattern($pattern)
```

Comment:
Add param and return types

Reply:
No you can't: https://3v4l.org/kCOA5

Reply:
Ok, maybe it's the other way round. All good. I checked and the interface is implemented by other classes as well and changing it would be a BC anyway.. so a thing for Matomo 6 perhaps. Thanks!